tests/integration/test_integration.py

```diff
@@ -747,19 +748,14 @@ def test_downgrade(encoding, monkeypatch):
 
 @allencodings
 def test_span_tags(encoding, monkeypatch):
-    monkeypatch.setenv("DD_TRACE_API_VERSION", encoding)
-
-    t = Tracer()
-    with mock.patch("ddtrace.internal.writer.writer.log") as log:
-        s = t.trace("operation", service="my-svc")
+    with snapshot_context(token="tests.integration.test_integration.test_span_tags"):
```

Comment:
Using `snapshot_context` allows us to generate one snapshot file for all encodings. Using `pytest.mark.snapshot(token=....)` did not work.

Reply:
ah, because otherwise we get one per-encoding... yeah... smart using `snapshot_context`